gamecenter/api/views.py

```diff
@@ -55,12 +56,48 @@ def signup():
 
 
 @blueprint.route('/leaderboards', methods=['GET', 'POST'])
+@handle_api_key
 @get_request_args
 def leaderboards_controller(args):
     if request.method == 'GET':
         return get_paginated_scores(args)
     else:
-        return create_entry()
+        return jsonify(data=SCORESCHEMA.dump(create_entry()).data)
+
+
+@blueprint.route('/add_score', methods=['POST'])
```

Comment:
This [isn't RESTful](https://en.wikipedia.org/wiki/Representational_state_transfer#Applied_to_web_services). If we're not going for being RESTful, that's fine, but we may need to take that out of the paper, or we can change our routes & docs